packages/lesswrong/lib/collections/tags/newSchema.ts

```diff
@@ -549,6 +549,7 @@ const schema = {
           ...getDefaultViewSelector("Comments"),
           tagId: tag._id,
           score: { $gt: 0 },
+          draft: false,
```

Comment:
Is this needed given the updated `defaultView`?  (It is a little weird/unfortunate that we aren't passing along the resolver context through `getDefaultViewSelector`, but that behavior works out to excluding draft comments by default anyways, so in this case it doesn't matter.)